run.py

```diff
@@ -88,12 +99,29 @@ def sub_menu_open(menu_entries, active_route):
 app.jinja_env.add_extension('jinja2.ext.loopcontrols')
 
 
-# Add datetime format filter
 def date_time_format(value, fmt='%Y-%m-%d %H:%M:%S'):
+    """
+    Add datetime format filter.
+
+    :param value: date
+    :type value: datetime
+    :param fmt: format for the returned string, defaults to '%Y-%m-%d %H:%M:%S'
+    :type fmt: str, optional
+    :return: string representing date-time in given format
+    :rtype: str
```

Comment:
Will be be able to drop the type & rtype if we switch to typing?

Reply:
Yes, we can drop `type` if we use typing but then if we generate documentation using some service it won't take into account the same and hence having both is redundant but better.